packages/sg721/src/lib.rs

```diff
@@ -92,31 +95,26 @@ pub struct UpdateCollectionInfoMsg<T> {
 }
 
 #[cw_serde]
-pub struct RoyaltyInfo {
-    pub payment_address: Addr,
+pub struct RoyaltyInfo<T: AddressLike> {
+    pub payment_address: T,
```

Comment:
How is Addr stored? as string? I guess is fully compatible .

Reply:
Think Shane addressed this